crates/aptos/e2e/main.py

```diff
@@ -118,28 +121,29 @@ def main():
 
     # Run a node + faucet and wait for them to start up.
     container_name = run_node(args.base_network, args.image_repo_with_project)
-    wait_for_startup(container_name, args.base_startup_timeout)
-
-    # Create the dir the test CLI will run from.
-    shutil.rmtree(args.working_directory, ignore_errors=True)
-    pathlib.Path(args.working_directory).mkdir(parents=True, exist_ok=True)
-
-    # Build the RunHelper object.
-    run_helper = RunHelper(
-        host_working_directory=args.working_directory,
-        image_repo_with_project=args.image_repo_with_project,
-        image_tag=args.test_cli_tag,
-        cli_path=args.test_cli_path,
-    )
-
-    # Prepare the run helper. This ensures in advance that everything needed is there.
-    run_helper.prepare()
-
-    # Run tests.
-    run_tests(run_helper)
-
-    # Stop the node + faucet.
-    stop_node(container_name)
+    try:
```

Comment:
Should we catch it as well, so we know what went wrong?

Reply:
We'll see what went wrong from the exception being raised, same as before. It's just this way the stop_node function runs unconditionally.

```
>>> try:
...     raise RuntimeError("hey")
... finally:
...     print("finally")
...
finally
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
RuntimeError: hey
```